fuels-signers/src/lib.rs

```diff
@@ -54,7 +51,7 @@ mod tests {
         let secret =
             SecretKey::from_slice(&secret_seed).expect("Failed to generate random secret!");
 
-        let provider = Provider::new(setup_local_node(vec![]).await);
+        let (provider, _) = setup_test_provider(vec![]).await;
```

Comment:
I had to change the `setup_test_provider` to also return the `SocketAddr` of the launched server, otherwise I needed to add tons of dependencies in the `fuels-abigen-macro` manifest (for the harness). I didn't explain it though.

Reply:
Can you show me some cases where I would call `setup_test_provider()` and actually use the `SocketAddr`? I couldn't find any in the tests cases here and I'm struggling to find a use case for it 😅.